src/vs/workbench/contrib/terminal/browser/media/shellIntegration-fish.fish

```diff
@@ -0,0 +1,91 @@
+# ----------------------------------------------------------------------------
```

Comment:
The files in that folder are a bit of a mess currently, but let's call this one just `shellIntegration.fish` to align with `shellIntegration-rc.zsh`/`shellIntegration.ps1`